cmd/awscollector/Dockerfile

```diff
@@ -1,10 +1,44 @@
-FROM alpine:latest as certs
+################################
+#	Building Stage         #	
+#			       #	
+################################
+FROM alpine:latest as build 
+
+# update cert
 RUN apk --update add ca-certificates
 
+# install libs
+RUN apk add --update bash git make build-base
+
+# config go
+COPY --from=golang:1.15.3-alpine /usr/local/go/ /usr/local/go/
+ENV PATH="/usr/local/go/bin:${PATH}"
+
+# copy artifacts
+WORKDIR /workspace
+COPY cmd  /workspace/cmd
+COPY config.yaml /workspace/config.yaml
+COPY pkg /workspace/pkg
+COPY tools /workspace/tools
+COPY go.mod /workspace/go.mod
+COPY go.sum /workspace/go.sum
+COPY VERSION /workspace/VERSION
+COPY .git /workspace/.git
```

Comment:
is it needed?

Reply:
yes, it's needed, we need .git to get latest commit hash, version to get version